templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -77,6 +77,32 @@ contents:
           mountPath: /etc/ssl/etcd/
         - name: sa
           mountPath: /var/run/secrets/kubernetes.io/serviceaccount/
+      - name: wait-for-membership
```

Comment:
this is kind of long how about just `membership` or `scale` or ? less to type :)